src/main/java/com/doit/api/Item/ItemController.java

```diff
@@ -0,0 +1,44 @@
+package com.doit.api.Item;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "/items")
+public class ItemController {
+    private ItemService itemService;
+
+    @Autowired
+    public ItemController(ItemService itemService) {
+        this.itemService = itemService;
+    }
+
+    @PostMapping(path = "/post")
+    public ResponseEntity<String> postItem(@RequestBody ItemRequest itemRequest) {
+        String response = itemService.addItem(itemRequest);
+        if (response == "ADDED")
+            return new ResponseEntity<>("ADDED", HttpStatus.OK);
+        else if (response == "CONFLICT")
+            return new ResponseEntity<>("Item already exists", HttpStatus.CONFLICT);
+        else
+            return new ResponseEntity<>("Item content too large", HttpStatus.PAYLOAD_TOO_LARGE);
+    }
+
+    @GetMapping(path = "/all")
+    public ResponseEntity<List<Item>> getItems() {
+        return new ResponseEntity<>(itemService.getItems(), HttpStatus.OK);
+    }
+
+    @GetMapping(path = "/{itemId}")
+    public ResponseEntity<Item> getItemById(@PathVariable long itemId) {
+        Item item = itemService.getItem(itemId);
+        if (item != null)
+            return new ResponseEntity<>(item, HttpStatus.OK);
+        else
+            return new ResponseEntity<>(item, HttpStatus.NOT_FOUND);
```

Comment:
Please don't omit the curly braces (`{ }`) even if the if statement contains only one instruction. Also, we don't need the  `else` in this case since we return in the `if`.